spikeextractors/extractors/bindatrecordingextractor/bindatrecordingextractor.py

```diff
@@ -17,18 +17,21 @@ class BinDatRecordingExtractor(RecordingExtractor):
         {'name': 'dtype', 'type': 'np.dtype', 'title': "The dtype of underlying data"},
         {'name': 'recording_channels', 'type': 'list', 'value':None, 'default':None, 'title': "List of recording channels"},
         {'name': 'frames_first', 'type': 'bool', 'value':True, 'default':True, 'title': "Frames first"},
-        {'name': 'offset', 'type': 'int', 'value':0, 'default':0, 'title': "Offset in binary file"},
-        {'name': 'probe_path', 'type': 'str', 'title': "Path to probe file (csv or prb)"},
+        {'name': 'geom', 'type': 'list', 'title': "list or array with channel locations"},
+        {'name': 'offset', 'type': 'int', 'value': 0, 'default': 0, 'title': "Offset in binary file"},
```

Comment:
I think we should change offset to offset_in_file to not mixin gain/offset in amplitude and the one of byte start.